apps/researcher/src/messages/nl/licence.mdx

```diff
@@ -0,0 +1,6 @@
+Dit betekent dat de informatie:
+
+- mag worden gekopieerd, verspreid, of doorgegeven via elk medium of bestandsformaat voor alle doeleinden, inclusief commerciële doeleinden.
+- mag worden bewerkt, geremixt, veranderd, of dat er voor alle doeleinden, inclusief commerciële doeleinden, afgeleide werken van gemaakt mogen worden.
+
+Gebruiker moeten de community als maker van het werk vermelden , een link naar de licentie plaatsen en aangeven of het werk veranderd is. Veranderingen moeten kenbaar gemaakt worden en mogen niet de indruk wekken dat de community ermee instemt.
```

Comment:
1. `Gebruikers` (similar to the English text)?
2. `vermelden, een` (remove space)